scripts/regenerate.sh

```diff
@@ -15,16 +15,16 @@
 
 set -eu -o pipefail
 
-WORKDIR=$(mktemp -d)
+export WORKDIR=$(mktemp -d)
```

Comment:
This will be used by protoc_installer.sh script as regenerate script will get triggered first and then protoc_installer, hence WORKDIR will be used there. 
Other option is to pass as an argument.

Reply:
That wasn't obvious to me. I think there should just be on way of installing protoc (right now there is one way vet.sh does and another way from regenerate.sh).